wandb/cli/cli.py

```diff
@@ -650,7 +650,31 @@ def _clean():
 @click.option("--program", default=False, help="Set sweep program")
 @click.option("--settings", default=False, help="Set sweep settings", hidden=True)
 @click.option("--update", default=None, help="Update pending sweep")
-@click.argument("config_yaml")
+@click.option(
+    "--stop",
+    is_flag=True,
+    default=False,
+    help="Finish a sweep to stop running new runs and let currently running runs finish.",
```

Comment:
```suggestion
    help="Stop a sweep: stop running new runs and let currently running runs finish.",
```

Reply:
The descriptions are copied as such from the UI. Are you sure?

Reply:
I think it would be helpful in these docstrings to add a bit more detail about what happens at the agent level with each of the various commands. I think that would be helpful for users who want to know exactly what will happen when a command is issued (rather than just the effects). So maybe we could rewrite the above as something like:

Stop a sweep. This command instructs the sweep agent process to exit after all currently running runs finish.